test-specifications/token-service/tokenCreateTransaction.md

```diff
@@ -0,0 +1,1150 @@
+
+#### JSON Request Example
+
+```json
+{
+  "jsonrpc": "2.0",
+  "id": 8895,
+  "method": "createToken",
+  "params": {
+    "name": "testname",
+    "symbol": "testsymbol",
+    "treasuryAccount": "0.0.2"
+  }
+}
+```
+
+#### JSON Response Example
+
+```json
+{
+  "jsonrpc": "2.0",
+  "id": 8895,
+  "result": {
+    "tokenId": "0.0.6415",
+    "status": "SUCCESS"
+  }
+}
+```
+
+### **Admin Key:**
+
+- The key which can perform administrative operations (update/delete) on the token.
+
+| Test no | Name                                                                                                             | Input                                                                                                                                                                                                 | Expected response                                                                                  | Implemented (Y/N) |
+|---------|------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------|-------------------|
+| 1       | Creates a token with a valid ED25519 public key as its admin key                                                 | name="testname", symbol="testsymbol", treasuryAccount=<OPERATOR_ACCOUNT_ID>, adminKey=<VALID_ED25519_PUBLIC_KEY>, commonTransactionParams.signers=[<CORRESPONDING_VALID_ED25519_PRIVATE_KEY>]         | The token creation succeeds and the token has the new ED25519 public key as its admin key.         | N                 |
+| 2       | Creates a token with a valid ECDSAsecp256k1 public key as its admin key                                          | name="testname", symbol="testsymbol", treasuryAccount=<OPERATOR_ACCOUNT_ID>, adminKey=<VALID_ECDSA_SECP256K1_PUBLIC_KEY>, commonTransactionParams.signers=[<CORRESPONDING_VALID_ED25519_PRIVATE_KEY>] | The token creation succeeds and the token has the new ECDSAsecp256k1 public key as its admin key.  | N                 |
+| 3       | Creates a token with a valid ED25519 private key as its admin key                                                | name="testname", symbol="testsymbol", treasuryAccount=<OPERATOR_ACCOUNT_ID>, adminKey=<VALID_ED25519_PRIVATE_KEY>, commonTransactionParams.signers=[<VALID_ED25519_PRIVATE_KEY>]                      | The token creation succeeds and the token has the new ED25519 private key as its admin key.        | N                 |
```

Comment:
You cannot use a private key as a key in Hedera.  Only _public_ keys are _ever_ sent to the network.